recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "1.0.1" %}
+{% set name = "glue-core" %}
+{% set version = "1.2.4" %}
```

Comment:
There are issues with the latest version.  Looking at this deeply, there are rapid and many changes.  Think it would be best to create the 1.2.4 version which is the last version before the project starts breaking API's and then jump to 1.5.0.